recipes/msgpack23/v3.0/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conan import ConanFile
```

Comment:
Why is this new version on a different recipe? Seems like the old one is identical to this (As far as I can see!) so we could reuse it just fine

Reply:
The sample project is different. As the old version one will not compile. And I figured that's the way to make the change without loosing v2.1.

Reply:
I've made the necessary changes to use the same recipe with different code for the test package, thanks for the info!